impl/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>StarRocks-Kettle-Connector</artifactId>
+        <groupId>org.pentaho.di.plugins</groupId>
+        <version>9.5.0.0-240</version>
+    </parent>
+
+    <artifactId>StarRocks-Kettle-Connector-impl</artifactId>
+    <name>PDI StarRocks Kettle Connector Impl</name>
+
+    <properties>
+        <slf4j.version>1.7.32</slf4j.version>
+        <log4j.version>2.17.2</log4j.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.starrocks</groupId>
+            <artifactId>starrocks-stream-load-sdk</artifactId>
+            <version>1.0-SNAPSHOT</version>
+            <classifier>jar-with-dependencies</classifier>
+        </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
```

Comment:
Not recommend to package the jdbc driver by default. Refer to [spark connector](https://docs.starrocks.io/zh-cn/latest/loading/Spark-connector-starrocks#110-%E5%8D%87%E7%BA%A7%E8%87%B3-111)